<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="cardLabel">
       <property name="text">
        <string>Enter Bumber of Cards</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cardLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="playerLabel">
       <property name="text">
        <string>Choose Number of Players  </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="playerHorizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="okPushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>500</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Players Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tableWidget</receiver>
   <slot>insertRow(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playerHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tableWidget</receiver>
   <slot>insertRow(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
